programs/zstdcli.c

```diff
@@ -947,6 +947,8 @@ int main(int argCount, const char* argv[])
         for (u=0, fileNamesNb=0; u<filenameIdx; u++) {
             if (UTIL_isLink(filenameTable[u])) {
                 DISPLAYLEVEL(2, "Warning : %s is a symbolic link, ignoring\n", filenameTable[u]);
+                if (filenameIdx == 1)
```

Comment:
We'll still be open to the same bug if we have 2 files which are both symlinks.

Reply:
Yes, your feedback makes sense. Commit received an update.
I have updated the test case for symbolic links, and have enabled it for OpenBSD.

Tested OK on OpenBSD.